apps/cookbook/src/app/examples/form-field-example/form-field-example-configuration-component/form-field-example-configuration.component.html

```diff
@@ -0,0 +1,26 @@
+<fieldset>
+  <legend>Input field size for examples below</legend>
+  <p>
+    <strong>Size:</strong><br />
+    <label>
+      <input
+        type="radio"
+        name="size"
+        value="md"
+        [checked]="size === 'md'"
+        (change)="sizeChanged.emit($event.target.value)"
+      />
+      Medium (<code>md</code>)
+    </label>
+    <label>
+      <input
+        type="radio"
+        name="size"
+        value="lg"
+        [checked]="size === 'lg'"
+        (change)="sizeChanged.emit($event.target.value)"
+      />
+      Large (<code>lg</code>) - default
+    </label>
```

Comment:
Consider using `kirby-radio-group`

Reply:
Curveball that one :) But makes perfectly sense.